build/docker/package.sh

```diff
@@ -4,11 +4,18 @@ set -euxo pipefail
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 VERSION=1.25.0
 
-mkdir -p $DIR/../../deploy/linux
-cp $DIR/../../integrations.json $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/
-cp $DIR/../../build/artifacts/createLogPath.sh $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/
+cd "$DIR/../.."
+mkdir -p deploy/linux
+cp integrations.json src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/
+cp build/artifacts/createLogPath.sh src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/
 
-cd $DIR/../../deploy/linux
+mkdir -p src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/netstandard2.0
+cp src/bin/windows-tracer-home/netstandard2.0/*.dll src/Datadog.Trace.ClrProfiler.Native/bin/Release/x64/netstandard2.0/
```

Comment:
Not necessarily in this PR, but we should look into porting some of these fixes: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/77/ instead of hard-coding `Release`.

Reply:
@andrewlock if you merge, I can apply my changes from #1309 to this PR as well. That PR was a port of the OTel PR Lucas mentioned

Reply:
Thanks. I forgot you did this already (and I even approved it! 😅)

Reply:
We're getting a lot of real integration test failures now, not sure if we want to merge this yet...

Reply:
Yeah I'll kick the build a little today since it's one of those trigger and come back sort of things

Reply:
@andrewlock looks like we're good now 👍🏼 Feel free to merge!